test/it/live-connect.spec.js

```diff
@@ -34,55 +34,58 @@ describe('LiveConnect', function () {
         ajaxTimeout: 3000
       }
     })
+    // wait for environment to become ready
+    await server.openPage('bln.test.liveintent.com', 'empty')
```

Comment:
turns out `window.localstorage` will be undefined until the first page load.
As we are probing localstorage before the first page load not doing this will cause the first test to always fail